sawmill-core/src/main/java/io/logz/sawmill/processors/AddSignatureProcessor.java

```diff
@@ -32,12 +36,14 @@ public ProcessResult process(Doc doc) throws InterruptedException {
         try {
             signature = createSignature(doc);
         } catch (Exception e) {
-            return ProcessResult.failure(
-                    "failed to create signature, SignatureMode: " + signatureMode,
+            String errorMessage = "failed to create signature, SignatureMode: " + signatureMode;
+            logger.debug(errorMessage + ", Exception: " + JsonUtils.toJsonString(e.getStackTrace()));
```

Comment:
![Screen Shot 2022-07-31 at 16 15 07](https://user-images.githubusercontent.com/46896871/182028182-79cc2c40-1be9-4397-b6fa-2f8c28fc3b5c.png)
No need to parse exception Trace yourself